ansible/roles/owncloud/tasks/setup_owncloud.yml

```diff
@@ -158,8 +158,8 @@
     name: 'ownCloud Background Jobs'
     minute: '{{ owncloud__cron_minute }}'
     user: '{{ owncloud__app_user }}'
-    job: 'test -x /usr/bin/php && test -e {{ (owncloud__deploy_path + "/cron.php") | quote }}
-          && /usr/bin/php -f {{ (owncloud__deploy_path + "/cron.php") | quote }}'
+    job: 'test -x /usr/bin/php && test -e {{ (owncloud__deploy_path + "/occ") | quote }}
+          && /usr/bin/php {{ (owncloud__deploy_path + "/occ") | quote }} system:cron'
     cron_file: 'owncloud'
```

Comment:
This does not work for Nextcloud. Can you make this change conditional based on if Nextcloud or ownCloud is used?

Ref: https://docs.nextcloud.com/server/17/admin_manual/configuration_server/background_jobs_configuration.html

Reply:
Done